runelite-client/src/main/java/net/runelite/client/plugins/thieving/ThievingConfig.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2018, Joris K <kjorisje@gmail.com>
+ * Copyright (c) 2018, Lasse <cronick@zytex.dk>
+ * Copyright (c) 2019, ermalsh <github.com/ermalsh>
+ * Copyright (c) 2019, Deadeye <github.com/DeadeyeOo>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.thieving;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("thieving")
+public interface ThievingConfig extends Config
+{
+	@ConfigItem(
+			position = 1,
+			keyName = "statTimeout",
+			name = "Reset stats (minutes)",
+			description = "Change the time until the thieving session is reset and the overlay is hidden"
+	)
+	default int statTimeout() { return 5; }
+
+	@ConfigItem(
+			position = 2,
+			keyName = "stunnedtimer",
+			name = "Stunned timer",
+			description = "Toggles timer when stunned"
+	)
+	default boolean stunedTimer() { return true; }
+
+	@ConfigItem(
+			position = 3,
+			keyName = "totalattempts",
+			name = "Total attempts",
+			description = "Toggles total thieving attempts"
+	)
+	default boolean totalAttempts()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			position = 4,
+			keyName = "successrate",
+			name = "Success Rate",
+			description = "Toggles percentage of successful thieving"
+	)
+	default boolean successRate()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			position = 5,
+			keyName = "skillperhour",

```

Comment:
This is pretty vague, I'd suggest something along the lines of "Successful attempts per hour"